cltk/corpus/sanskrit/corpora.py

```diff
@@ -2,4 +2,8 @@
         {'name':'sanskrit_text_jnu',
         'location':'remote',
         'type':'text'},
+        {'name':'sanskrit_text_dcs',
+        'location':'remote',
+        'type':'text'
+            },
```

Comment:
That , will not create a problem. You can see that in other directory's corpora.py file of this repo(like greek, latin) Thank you @nishnik